### ISSUE TITLE
Keep html comments after applying releaseTheKraken

### ISSUE THREAD

**Issue by ovidals** [foundation/foundation-emails#537]
Hi,

is there any way to keep html comments after applying the "releaseTheKraken" function?

Thank you.

**Comment by kball** [comment]
Have you tried embedding them in a `<raw>` tag? That will hide them from inky, though it may be that the css inliner will also strip them out... I'm working on making it so the raw tag will also hide things from the inliner, so if you try and they still get ripped out, that will help in a bit.

**Comment by ovidals** [comment]
Yes, the raw doesn't prevent the strip with inky + inliner. I'm working with twig too... Isn't there any option so?

Thank you.

**Comment by DanielRuf** [comment]
We would need a reproducible testcase to see if this still occurs and what we have to fix.

**Comment by joeworkman** [comment]
Foundation for Emails has been superseded by [Inky v2](https://github.com/foundation/inky), a complete rewrite in Rust with improved Outlook support, hybrid output mode, bulletproof buttons, validation, and more. This issue will not be addressed in the legacy codebase.

If this issue persists in v2, please open a new issue at https://github.com/foundation/inky/issues
